### PR TITLE
Retry Elasticsearch cluster healthcheck

### PR DESCRIPTION
**Pull request by pschoffer**
We currently don't retry on failed ES check, which beats the purpose of the while loop.

### CODE REVIEW

File: dev-tools/setup.sh [hunk 1]
```diff
@@ -126,7 +126,7 @@ cp /dev-tools/dev-env-plugin.php /wp/wp-content/mu-plugins/
 if [ -n "${ENABLE_ELASTICSEARCH}" ] || { [ -n "${LANDO_INFO}" ] && [ "$(echo "${LANDO_INFO}" | jq .elasticsearch.service)" != 'null' ]; }; then
   echo "Waiting for Elasticsearch to come online..."
   second=0
-  while ! curl -s 'http://elasticsearch:9200/_cluster/health' > /dev/null; do
+  while ! curl -s 'http://elasticsearch:9200/_cluster/health' > /dev/null && [ "${second}" -lt 60 ]; do
```
Comment (rebeccahum) [inline dev-tools/setup.sh:129]: Do we want a limit to the retries?

**Review by rebeccahum — CHANGES_REQUESTED** [review]

File: dev-tools/setup.sh (hunk 1 above)
Comment (sjinks) [reply dev-tools/setup.sh:129]: Yes, I missed that part.

Comment (sjinks) [reply dev-tools/setup.sh:129]: It's like https://github.com/Automattic/vip-container-images/blob/91ad5f152d20a031a23bad9ab29d2fa73275f501/dev-tools/setup.sh#L104-L107

**Review by sjinks — APPROVED** [review]

**Comment by pschoffer** [comment]
I added a dot-adder to show it is in progress.

**Review by sjinks — APPROVED** [review]

**Pull request merged**